src/main/scala/exercises/04_Options.scala

```diff
@@ -34,7 +34,15 @@ object Options {
    * - does not exist: 					"not existing"
    */
   def roomState(rooms: Map[Int, Option[String]], room: Int): String = {
-    error("Fix me")
+  	if (!rooms.contains(room)) "not existing" else rooms.getOrElse(room, Some("empty")).getOrElse("")
```

Comment:
I have yet to try using the getOrElse method, but it seems like an elegant alternative to pattern matching! Wondering if your function handles the case of a locked room though.